### PR TITLE
Created feature-branch1 with change6

### PR DESCRIPTION
**Pull request by ak-c**

### CODE REVIEW

File: read-me.txt [hunk 1]
```diff
@@ -6,3 +6,4 @@ Added for change3
 Created remote repo. Pushed it. This will be first commit after linking local to remote change4
 Editing this change in the github browser remotely - to test pull command
 After adding ak-c as collaborator. Changes added by ak-c change5
+Created feature-branch1 : added initial changes bc:1 c6
```
Comment (ak-m) [inline read-me.txt]: change the verbiage initial changes to initial-changes

**Review by ak-m — CHANGES_REQUESTED** [review]
added the comments - please make those changes

File: read-me.txt (hunk 1 above)
Comment (ak-c) [reply read-me.txt]: Ok will make changes and commit

Comment (ak-m) [reply read-me.txt]: Looks good now accepting those changes

**Pull request merged**